pint/testsuite/test_numpy.py

```diff
@@ -758,9 +768,12 @@ def test_minimum(self):
             np.minimum(self.q, self.Q_([0, 5], "m")), self.Q_([[0, 2], [0, 4]], "m")
         )
 
+    # NP2: Can remove Q_(arr).ptp test when we only support numpy>=2
     def test_ptp(self):
-        assert self.q.ptp() == 3 * self.ureg.m
+        if not np.lib.NumpyVersion(np.__version__) >= "2.0.0b1":
```

Comment:
I would probably use `@pytest.mark.skipif(np.lib.NumpyVersion(np.__version__) >= "2.0.0b1, reason="removed in numpy2")` instead here

Reply:
agree